### ISSUE TITLE
Whatsapp Config Tools, not working anymore with new version Wa keystore.xml

### ISSUE THREAD

**Issue by bahtiarp** [lovethiscode/Gorgeous-Whatsapp#43]
Whatsapp Config Tools, did not working anymore with new version Wa keystore.xml
Do you have new version of Whatsapp Config Tools that works also with : new type of keystore.xml
sample new keystore.xml:
`<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
<map>
    <long name="client_static_keypair_enc_success" value="501" />
    <boolean name="can_user_android_key_store" value="true" />
    <int name="client_android_key_store_auth_ver" value="1" />
    <string name="client_static_keypair_enc">[0,&quot;7jMr9M9P7B+eogYwGrPTZBDcUrIUZ\/pb+I4Zf31\/Hz0X+JVGW+HUUR\/kiaLaWY5R0NVLwl7VZMniuZuqbmPd6yknQq1OfNizisnpJWUSGB4&quot;,&quot;LlodYP1kLh14PjzY&quot;]</string>
    <string name="server_static_public">xDn6MqBPn3O6ptDhPQt/tqcXrv2dK7aR//NQLFIVal0</string>
</map>
`

**Comment by lovethiscode** [comment]
I'm not maintaining this project anymore because of the busy working hours.



发自我的iPhone


------------------ Original ------------------
From: bahtiarp ***@***.***&gt;
Date: Mon,Nov 8,2021 8:51 AM
To: lovethiscode/Gorgeous-Whatsapp ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [lovethiscode/Gorgeous-Whatsapp] Whatsapp Config Tools, not workinganymore with new version Wa keystore.xml (Issue #43)





 
Whatsapp Config Tools, did not working anymore with new version Wa keystore.xml
 Do you have new version of Whatsapp Config Tools that works also with : new type of keystore.xml
 sample new keystore.xml:
 <?xml version='1.0' encoding='utf-8' standalone='yes' ?&gt; <map&gt; <long name="client_static_keypair_enc_success" value="501" /&gt; <boolean name="can_user_android_key_store" value="true" /&gt; <int name="client_android_key_store_auth_ver" value="1" /&gt; <string name="client_static_keypair_enc"&gt;[0,&amp;quot;7jMr9M9P7B+eogYwGrPTZBDcUrIUZ\/pb+I4Zf31\/Hz0X+JVGW+HUUR\/kiaLaWY5R0NVLwl7VZMniuZuqbmPd6yknQq1OfNizisnpJWUSGB4&amp;quot;,&amp;quot;LlodYP1kLh14PjzY&amp;quot;]</string&gt; <string name="server_static_public"&gt;xDn6MqBPn3O6ptDhPQt/tqcXrv2dK7aR//NQLFIVal0</string&gt; </map&gt; 
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by lovethiscode** [comment]
Now it needs a new method to decrypt

**Comment by bahtiarp** [comment]
> Now it needs a new method to decrypt

do you have a clue how to decrypt for the new mothod?

**Comment by lovethiscode** [comment]
Yes, I can decrypt it, but I won't disclose the decryption tool.



发自我的iPhone


------------------ Original ------------------
From: bahtiarp ***@***.***&gt;
Date: Sun,Nov 14,2021 10:51 PM
To: lovethiscode/Gorgeous-Whatsapp ***@***.***&gt;
Cc: lovethiscode ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [lovethiscode/Gorgeous-Whatsapp] Whatsapp Config Tools, notworking anymore with new version Wa keystore.xml (Issue #43)





  
Now it needs a new method to decrypt
  
do you have a clue how to decrypt for the new mothod?
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by bahtiarp** [comment]
do you have the paid version?

**Comment by lovethiscode** [comment]
Add my telegram,
gorgeous008






------------------ Original ------------------
From: bahtiarp ***@***.***&gt;
Date: Mon,Nov 15,2021 9:15 AM
To: lovethiscode/Gorgeous-Whatsapp ***@***.***&gt;
Cc: lovethiscode ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [lovethiscode/Gorgeous-Whatsapp] Whatsapp Config Tools, notworking anymore with new version Wa keystore.xml (Issue #43)





 
do you have the paid version?
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by alex1491163117** [comment]
TG消息不能发送，需要加好友哦

**Comment by huoxing007** [comment]
> TG消息不能发送，需要加好友哦

gorgeous006

**Comment by lovethiscode** [comment]
> TG消息不能发送，需要加好友哦

gorgeous006
